setup.Rmd

```diff
@@ -13,52 +13,36 @@ install.packages(c("devtools", "roxygen2", "testthat", "knitr"))
 ```
 
 Make sure you have a recent version of the RStudio integrated development environment (IDE).
-In fact, consider using the preview version and updating regularly.
-Compared to the official released version, the preview gives you access to the latest and greatest features and only slightly increases your chances of finding a bug.
-It is distinct from the more volatile daily build.
+New versions are released regularly, so we recommend updating often to get access to the latest and greatest features.
 
--   Preview version: <https://www.rstudio.com/products/rstudio/download/preview/>
--   Released version: <https://www.rstudio.com/products/rstudio/download/>
-    -   Most readers can use the free, open source version of RStudio Desktop.
+Download the current version of RStudio Desktop here: <https://posit.co/download/rstudio-desktop/>.
+Most readers can use the free, open source version of RStudio Desktop.
```

Comment:
Yeah, this is a good edit. Just to provide you with more background, here's an excerpt from a related slack conversation, when we made similar changes to the pre-workshop setup instructions for WTF. These quotes are from folks in our support organization, confirming my sense that the preview version is much less relevant than in the past:

> ... the Preview release is usually the release candidate for the next general release. It’s not updated often - just before a new release, when the engineering team thinks they have a good candidate. By definition, the Preview release will match the production release until the following release has a candidate, usually a few weeks at most before the next release

> We are definitely releasing more often than we used to, and we therefore put out fewer Previews between GA releases (bigger, longer release cycles mean there’s more risk for regressions and bugs that require new Previews before GA). It’s true that even when Preview and GA differ, there isn’t usually a big gap anymore, with the exception of releases with breaking changes (though those are rare).

> Previews have been utilized less with the change to shorter release cycles. We may have a preview up after "feature complete" for a couple of weeks before a release while we stabilize/certify the final build and that's it.